README.md

```diff
@@ -1,6 +1,18 @@
 # evidence
 
-Word2Vec-based assisted close reading tool with support for context-based search and concept-based search.
+Doc2Vec-based assisted close reading tool with support for abtract concept-based search and context-based search.
+
+## Machine supported research in humanities
+While research in the humanities has been able to leverage the digitization of text corpora and the development of computer based text analysis tools to its benefit, the interface current systems provide the user with is incompatible with the the proven method of scholarly close reading of texts which is key in many research scenarios pursuing complex research questions.
+
+What this boils down to, is the fact that it is often restrictive and difficult to formulate selection criteria, in particular for more complex or abstract concept, in the framework of a keyword based search which is the standard entry point to digitized text collections.
+
+## Querying by example - close reading with tailored suggestions  
+Evidence provides an alternative, intuitive entry point into collections. Using the doc2vec framework evidence learns abstract representations of the content of the elements of the user's corpus.
```

Comment:
One could mistake "Evidence" for its normal meaning. Can you think of a way to avoid this confusion?

Reply:
how about a different font type for the tool throughout? are there any conventions?
Or is that abhorrent to you?

Reply:
@jspaaks how about a different font type for the tool throughout? are there any conventions?
Or is that abhorrent to you?

Reply:
Whichever way you think is OK, I just wanted to raise the issue. Simplest way to resolve this is maybe just ``evidence``?

Reply:
I was thinking the same. I'll go for that